test/common/schemaform/fields/ArrayField.unit.spec.jsx

```diff
@@ -126,13 +129,13 @@ describe('Schemaform <ArrayField>', () => {
         {},
         {}
       ];
-      errorSchema = {};
+      fullErrorSchema = {};
```

Comment:
Is there a problem with using `errorSchema` here?

Reply:
It is declared in the upper scope.